docs/baking.md

```diff
@@ -220,25 +221,96 @@ In case you want to use a different alias for the baking account:
 2. update the `BAKER_ADDRESS_ALIAS` by editing the
     `/etc/default/tezos-baking-<network>` file.
 
-## Running baking instance on edo2net
+## Quick Start
 
-TL;DR, in order to run a baking instance on edo2net you should do the following:
+<details>
+ <summary>
+   <em>Optional</em> Create new Ubuntu virtual machine...
+ </summary>
 
-1) Install `tezos-baking` package following either [Ubuntu](#ubuntu) or [RaspberryPi OS](#pios)
-instructions.
+A quick way to spin up a fresh Ubuntu virtual machine is
+to use [Multipass](https://multipass.run/) (reduce disk if this is
+to be used with a test network or a mainnet node in rolling history mode):
 
-2) Run following commands:
 ```
-snapshot_file=/tmp/tezos-edo2net.rolling
-wget https://edo2net.xtz-shots.io/rolling -O "$snapshot_file"
-sudo -u tezos tezos-node-edo2net snapshot import "$snapshot_file"
+multipass launch --cpus 2 --disk 100G --mem 4G --name tezos
+```
 
-sudo systemctl start tezos-node-edo2net
+and then log in:
 
-sudo -u tezos tezos-client bootstrapped
-sudo -u tezos tezos-client import secret key baker <secret-key>
+```
+multipass shell tezos
+```
+
+> Note that on Windows and MacOS this VM will not have access to USB and
+> thus is not suitable for using with Ledger Nano S.
+
+</details>
+
+1) Install `tezos-baking` package following either [Ubuntu](#ubuntu)
+or [RaspberryPi OS](#pios) instructions.
+
+2) Choose one of supported Tezos networks - `mainnet`, `florencenet` or
+`edo2net` by setting environment variable:
+
+```
+tznet=florencenet
+```
+
+3) Initialize Tezos node from a snapshot and start node service:
+
+```
+snapshot_file=/tmp/tezos-$tznet.rolling
+snapshot_url=https://$tznet.xtz-shots.io/rolling
+wget $snapshot_url -O $snapshot_file
```

Comment:
The `snapshot_url` variable seems unnecessary, I suggest avoiding it:
```suggestion
wget "https://$tznet.xtz-shots.io/rolling" -O $snapshot_file
```

Reply:
the variable is not strictly necessary, but I think it improves readability and usability by giving a meaningful name to otherwise nameless url, making wget command shorter, hinting that the url itself replacable/pluggable and giving an opportunity to set/change it independently of actual download command

Reply:
Fair enough, those are good point :+1: